addons/l10n_vn/data/account_tax_report_data.xml

```diff
@@ -56,12 +56,23 @@
                                     </record>
                                 </field>
                             </record>
+                            <record id="account_tax_report_line_04_01_01_vn" model="account.report.line">
+                                <field name="name">VAT Exemption on purchase of goods and services</field>
+                                <field name="code">VAT_EXEMPTION_ON_PURCHASE_OF_GOODS_AND_SERVICES</field>
+                                <field name="expression_ids">
+                                    <record id="account_tax_report_line_04_01_01_vn_tag" model="account.report.expression">
+                                        <field name="label">balance</field>
+                                        <field name="engine">tax_tags</field>
+                                        <field name="formula">VAT Exemption on purchase of goods and services</field>
+                                    </record>
+                                </field>
+                            </record>
```

Comment:
same as previous comment.
I may be wrong but I think this part shouldn't be added as there isn't any vat with an "exemption tax"